src/_layouts/home.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en-US" prefix="og: http://ogp.me/ns#">
+<html lang="en-US" prefix="og: http://ogp.me/ns#" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
```

Comment:
worth looking at https://primer.style/css/support/theming as there's a few different `data-dark-theme` options, I personally preferred `dark` the most when briefly looking at them all